frontend/src/components/campAdmin/DialogMaterialListForm.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-  <div>
+  <e-form name="materialList">
     <e-text-field
       v-model="localMaterialList.name"
-      :name="$tc('entity.materialList.fields.name')"
+      name="name"
       autofocus
       vee-rules="required"
     />
-  </div>
+  </e-form>
 </template>
 
 <script>
+import EForm from '@/components/form/base/EForm.vue'
+
 export default {
   name: 'DialogMaterialListForm',
+  components: { EForm },
```

Comment:
Should we add EForm to the list of autoloaded components? Apparently all our E-input components are autoloaded, and according to the change you are doing here, pretty much every timewe use an E-input, we also use an EForm

Reply:
Agreed, fixed in b3b1397